src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/ApplicationTest.java

```diff
@@ -41,8 +44,7 @@ void runErrorProcessingTrueTest() {
 
         verify(taskScheduler, times(0)).scheduleWithFixedDelay(eq(mainMessageConsumer), anyLong());
         verify(taskScheduler, times(0)).scheduleWithFixedDelay(eq(retryMessageConsumer), anyLong());
-
-        //ToDo Ensure error consumer is called
+        verify(taskScheduler, times(1)).scheduleWithFixedDelay(eq(errorMessageConsumer), anyLong());
```

Comment:
I would put eq(1L) instead of anyLong() just to ensure the delay isnt longer than we'd expect